docs/Building-an-OpenSearch-and-OpenSearch-Dashboards-Distribution.md

```diff
@@ -129,3 +128,9 @@ For plugin artifacts, you can also use the `latest` keyword to get the latest pl
 For bundled artifacts, here are some examples for LINUX and Windows:
 * Linux Tar: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.4.0/latest/linux/x64/tar/dist/opensearch/opensearch-2.4.0-linux-x64.tar.gz
 * Windows Zip: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.4.0/latest/windows/x64/zip/dist/opensearch/opensearch-2.4.0-windows-x64.zip
```

Comment:
Same as above, can you use latest versions which utilizes the changes.

Reply:
Done